pom.xml

```diff
@@ -1543,67 +1543,80 @@
                 <exclude>**/DefaultConfigurationTest.class</exclude>
                 <!-- Inline Config cannot be supported. -->
                 <exclude>**/UniquePropertiesCheckTest.class</exclude>
+                <exclude>**/OrderedPropertiesCheckTest.class</exclude>
 
+                <!-- some tests are not for java files, need support of different header comment -->
+                <exclude>**/RegexpMultilineCheckTest.class</exclude>
                 <exclude>**/RegexpCheckTest.class</exclude>
 
-                <exclude>**/SuppressionSingleFilterTest.class</exclude>
-
+                <!-- test need to be updated -->
                 <exclude>**/AllSinglelineCommentsTest.class</exclude>
 
+                <!-- usage of custom internal module -->
                 <exclude>**/FileSetCheckTest.class</exclude>
-
                 <exclude>**/AbstractFileSetCheckTest.class</exclude>
 
-                <exclude>**/OrderedPropertiesCheckTest.class</exclude>
-
+                <!-- test needs to be updated -->
```

Comment:
Can we do standard `until https://github.com/checkstyle/checkstyle/issues/11446` now that we have open issue?

Reply:
Resolved.